### PR TITLE
Storage clear documentation

### LINKED ISSUE #2696: [Term Entry] JavaScript Storage .clear()
### Reason/inspiration (optional)

We would like a new term entry for JavaScript [Storage](https://www.codecademy.com/resources/docs/javascript/storage) about the .clear() method. The entry should go in a new file in docs/content/javascript/concepts/storage/terms/clear.md

The entry should
    - Have an introductory paragraph about what the method is and does.
    - A syntax section with a pseudo code block with an explanation of it.
    - An example showing the method in use.

Please refer to the [term entry templates](https://github.com/Codecademy/docs/blob/main/documentation/term-entry-template.md), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [ ] Issue is assigned

### PR DESCRIPTION
**Pull request by MB1721**
### Description

Iadded a new term to the Storage concept for Javascript. The change is located in content/javascript/concepts/storage/terms/clear.md. Closes #2696 

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/javascript/concepts/storage/terms/clear.md
Comment (SSwiniarski) [inline content/javascript/concepts/storage/terms/clear.md:3]:
```suggestion
Description: 'Removes all keys in a Storage object.'
```

File: content/javascript/concepts/storage/terms/clear.md
Comment (SSwiniarski) [inline content/javascript/concepts/storage/terms/clear.md:27]:
```suggestion
## Example
```

File: content/javascript/concepts/storage/terms/clear.md
Comment (SSwiniarski) [inline content/javascript/concepts/storage/terms/clear.md:28]:
```suggestion
In this example new items are added to `sessionStorage` and `localStorage`. The `.clear()` method then removes all data 
```

File: content/javascript/concepts/storage/terms/clear.md
Comment (SSwiniarski) [inline content/javascript/concepts/storage/terms/clear.md:31]:
````suggestion
```js
````

File: content/javascript/concepts/storage/terms/clear.md
Comment (SSwiniarski) [inline content/javascript/concepts/storage/terms/clear.md:51]:
````suggestion
```

 This example results in the following output:

```shell
Storage {length: 0}
Storage {length: 0}
```
````

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@MB1721 I did a first review and left a few comments.

File: content/javascript/concepts/storage/terms/clear.md
Comment (MB1721) [reply content/javascript/concepts/storage/terms/clear.md:27]: I was under the impression that codebyte examples required the "## Codebyte Example" header and did not require and "output" block? Are codebyte examples only used when a regular Example block is already present?

Comment (SSwiniarski) [reply content/javascript/concepts/storage/terms/clear.md]: The problem here is that the code as written doesn't work with the engine running the codebytes.

File: content/javascript/concepts/storage/terms/clear.md
Comment (SSwiniarski) [inline content/javascript/concepts/storage/terms/clear.md:46]:
```suggestion
// Add items to session storage.
sessionStorage.setItem('tab-item1', 'TI-1');
sessionStorage.setItem('tab-item2', 'TI-2');

// Add items to local storage.
localStorage.setItem('domain-item1', 'DI-1');
localStorage.setItem('domain-item2', 'DI-2');

// Remove all items from both local and session storage.
sessionStorage.clear();
localStorage.clear();

// Output storage.
console.log(sessionStorage.length);
console.log(localStorage.length);
```

File: content/javascript/concepts/storage/terms/clear.md
Comment (SSwiniarski) [inline content/javascript/concepts/storage/terms/clear.md:53]:
```suggestion
0
0
```

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@MB1721 I had a few more changes.

File: content/javascript/concepts/storage/terms/clear.md
Comment (MB1721) [reply content/javascript/concepts/storage/terms/clear.md]: Is there a way to check what the md file looks like once it goes online? I don't know how to display markdown files on my local machine without seeing them in github. Are there any programs that support markdown?

Comment (SSwiniarski) [reply content/javascript/concepts/storage/terms/clear.md]: > Is there a way to check what the md file looks like once it goes online? I don't know how to display markdown files on my local machine without seeing them in github. Are there any programs that support markdown?

@MB1721, there's a list here: https://becomeawritertoday.com/best-markdown-editor/

**Comment by MB1721** [comment]
Is this ready to be merged yet?

**Comment by SSwiniarski** [comment]
> Is this ready to be merged yet?

@MB1721 Be patient, the entry requires two maintainer reviews.

**Review by SSwiniarski — APPROVED** [review]
@MB1721 This is ready for a second review.

File: content/javascript/concepts/storage/terms/clear.md
Comment (caupolicandiaz) [inline content/javascript/concepts/storage/terms/clear.md:2]: A folder named clear should be added inside "terms". The path should be -
content/javascript/concepts/storage/terms/clear/clear.md

File: content/javascript/concepts/storage/terms/clear.md
Comment (caupolicandiaz) [inline content/javascript/concepts/storage/terms/clear.md:18]:
```suggestion
**`.clear()`** removes all keys in a `Storage` object.
```

File: content/javascript/concepts/storage/terms/clear.md
Comment (caupolicandiaz) [inline content/javascript/concepts/storage/terms/clear.md:20]: add an empty line after the syntax header as well as the example header

File: content/javascript/concepts/storage/terms/clear.md
Comment (caupolicandiaz) [inline content/javascript/concepts/storage/terms/clear.md:23]:
```suggestion
```

File: content/javascript/concepts/storage/terms/clear.md
Comment (caupolicandiaz) [inline content/javascript/concepts/storage/terms/clear.md:24]: add an empty line after the block

File: content/javascript/concepts/storage/terms/clear.md
Comment (caupolicandiaz) [inline content/javascript/concepts/storage/terms/clear.md:28]:
```suggestion
In this example new items are added to `sessionStorage` and `localStorage` from console. The `.clear()` method then removes all data from each `Storage` object.
```

File: content/javascript/concepts/storage/terms/clear.md
Comment (caupolicandiaz) [inline content/javascript/concepts/storage/terms/clear.md:29]:
```suggestion
```

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@MB1721 I've added my comments to your entry. Please update, and tag me when you've finished making changes. Thanks

File: content/javascript/concepts/storage/terms/clear.md
Comment (CBID2) [inline content/javascript/concepts/storage/terms/clear.md:28]: [Proofreading] Always place a comma after an introductory element. 
```suggestion
In this example, new items are added to `sessionStorage` and `localStorage` from console. The `.clear()` method then removes all data 
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @MB1721! :) I pointed out a typo that needs to be fixed. Other than that and @caupolicandiaz's suggestions, you're off to a pretty good start! :)

File: content/javascript/concepts/storage/terms/clear/clear.md
Comment (CBID2) [inline content/javascript/concepts/storage/terms/clear/clear.md:30]:
```suggestion
In this example, new items are added to `sessionStorage` and `localStorage` from console. The `.clear()` method then removes all data.
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Just found a typo @MB1721

**Comment by MB1721** [comment]
@CBID2 and @caupolicandiaz I have made the requested changes.

**Review by caupolicandiaz — APPROVED** [review]
@MB1721 @SSwiniarski LGTM

**Comment by SSwiniarski** [comment]
@MB1721 Congrats on the new entry. Live link here: https://www.codecademy.com/resources/docs/javascript/storage/clear

image

You're currently anonymous. If you don't wish to be anonymous, check out this forum thread on how to link your GitHub and Codecademy profiles: https://discuss.codecademy.com/t/my-contribution-is-live-but-im-not-being-shown-as-a-contributor-why-is-this/758036  **Note:** it may take a few minutes for the change to register on the entry page.

**Pull request merged**
